powertools-logging/src/test/java/software/amazon/lambda/powertools/logging/internal/LambdaLoggingAspectTest.java

```diff
@@ -172,6 +174,21 @@ void shouldLogServiceNameWhenEnvVarSet() throws IllegalAccessException {
                 .containsEntry("service", "testService");
     }
 
+    @Test
+    void shouldLogxRayTraceIdEnvVarSet() {
+        String xRayTraceId = "1-5759e988-bd862e3fe1be46a994272793";
+
+        try (MockedStatic<SystemWrapper> mocked = Mockito.mockStatic(SystemWrapper.class)) {
+            mocked.when(() -> SystemWrapper.getenv("_X_AMZN_TRACE_ID")).thenReturn("Root=1-5759e988-bd862e3fe1be46a994272793;Parent=53995c3f42cd8ad8;Sampled=1\"");
```

Comment:
@pankajagrawal16 did you have another way of doing this?

Reply:
This was the way actually 🙂.. thats how we have done it in other modules